README.md

```diff
@@ -10,21 +10,24 @@ We are calling those new way of packaging Deep Learning models: __Rockets__.
 __Welcome to the Rockets Scientists Community!!!__
 
 ## Install the repositories
-We recommend you to use an isolated Python environement such as [virtualenv](https://virtualenv.pypa.io/en/latest/) or [conda](https://docs.conda.io/en/latest/) with at least __Python 3.5__. Then you can use the following lines of code:
+We recommend you to use an isolated Python environement such as [virtualenv](https://virtualenv.pypa.io/en/latest/) or [conda](https://docs.conda.io/en/latest/) with at least __Python 3.6__. Then you can use the following lines of code:
 ```
-git clone https://github.com/LucasVandroux/PyTorch-Rocket-YOLOv3-RetinaNet-SSD
-cd PyTorch-Rocket-YOLOv3-RetinaNet-SSD
-pip install -r requirements.txt
+git clone https://github.com/LucasVandroux/PyTorch-Rocket-YOLOv3-RetinaNet50-RetinaNet101
+cd PyTorch-Rocket-YOLOv3-RetinaNet50-RetinaNet101
+pip install rocketbase
```

Comment:
As rocketbase, contains all of the needed dependencies we don't need the requirements.txt anymore.